components/form/FormInput.tsx

```diff
@@ -0,0 +1,29 @@
+import clsx from 'clsx'
+import { TextInput, View } from 'react-native'
+
+interface Props {
+  valueInput: string
+  placeHolder: string
+  leftIcon?: JSX.Element
+  rightIcon?: JSX.Element
+}
+
+function FormInput({ valueInput, placeHolder, leftIcon, rightIcon }: Props) {
```

Comment:
cambiar nombre del componente a uno mas acorde al entorno mobile, en mobile no existe un componente form como en web que existe la etiqueta form, en su lugar : 
```suggestion
function Input({ value, placeholder, leftIcon, rightIcon }: Props) {
```